book-1-orientation/chapters/CLASSES_INTRO.md

```diff
@@ -1,27 +1,42 @@
 # User Defined Types
 
-A class is a blueprint, or a template, for creating an object instance in memory. C# and the .NET Framework define many types, such as `int`, `decimal`,  `bool`, `class`, `string`, and `interface`. In C#, you can also create your own, custom reference types.
+In a programming language a **type** is a blueprint, or a template, for creating an object instance in memory. A type defines the capabilities of an object. C# and the .NET Framework provide many types, such as `int`, `decimal`, `bool`, `Datetime`, `string`, `Dictionary`, `List`, etc... There are literally thousands of them!
 
-You create a new reference type with a class.
+While these types provide a lot of functionality, each C# application must also provide its own, custom types to do the specific work it needs to do.
+
+In C# the most common way to create a new type is with a `class`.
 
 > **Further reading:** [Types in C#](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/types)
 
 ```cs
 public class Writer
 {
-    public void write (string message)
+    public Writer(string prefix) {
+        Prefix = prefix;
+    }
+
+    public string Prefix {get; set;}
+
+    public void Write (string message)
     {
-        Console.WriteLine(message);
+        Console.WriteLine(Prefix + message);
     }
 }
 
 // The output variable's type is `string` -- a built in type
 string output = "Nashville Software School";
 
 // The author variable's type is Writer -- a custom type you defined
-Writer author = new Writer();
-author.write(output);
+Writer author = new Writer("I do declare: ");
+author.Write(output);
+
+author.Prefix = "That's right, I said: ";
+author.Write(output);
 ```
+The above code would print the following:
+
+    I do declare: Nashville Software School
+    That's right, I said:: Nashville Software School
```

Comment:
super pedantic, but there shouldn't be 2 colons printed.

Reply:
👍

Reply:
Thanks for the proofreading.

Unless I missed something I don't think access modifiers have been mentioned prior to this chapter. Personally, I feel like that part of this chapter needs some work, having never taught it, I don't have ideas as to what needs to be changed. I decided to focus on the changes I was more certain about.